android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/QuestionnaireConfig.kt

```diff
@@ -34,7 +34,8 @@ data class QuestionnaireConfig(
   val resourceType: String? = null,
   val confirmationDialog: ConfirmationDialog? = null,
   val groupResource: GroupResourceConfig? = null,
-  val taskId: String? = null
+  val taskId: String? = null,
+  val saveDraft: Boolean = false
```

Comment:
Do we expect this property to be dynamic? such that it can be computed via Rules engine?

Reply:
Hi @ellykits I don't expect so. I can't think of a scenario where we'll have this computed via Rules Engine. 
@dubdabasoduba Thoughts?

Reply:
Noted.